proposals/nnnn-mutex.md

```diff
@@ -11,18 +11,11 @@ This proposal introduces a mutual exclusion lock, or a mutex, to the standard li
 
 ## Motivation
 
-Since the dawn of Swift, one of the most difficult issues to resolve has been synchronizing access for core standard library collections such as `Array`, `Dictionary` and `Set`. These types are not concurrent and do not provide any sort of thread safety, and are prone to data races. For example, it is a common pitfall to implement a class that contains a property of type `Dictionary` without proper synchronization ensuring that access to this property is well defined across multiple threads.
+In concurrent programs, protecting shared mutable state is one of the core fundamental problems to ensuring reading and writing data is done in an explainable fashion. Synchronizing access to shared mutable state is not a new problem in Swift however. We've introduced many features to help protect mutable data. Actors are a straightforward solution to protecting your mutable state because it isolates the stored data in its own domain. At any one time, only one execution context will have exclusive access to this data. Multiple contexts cannot race with each other and leave your storage in an inconsistent or broken state. The actor model guarantees that all mutable state stored within it is isolated (except `nonisolated` and `let` storage) and operations that access such data are coupled together.
```

Comment:
```suggestion
In concurrent programs, protecting shared mutable state is one of the core fundamental problems to ensuring reading and writing data is done in an explainable fashion. Synchronizing access to shared mutable state is not a new problem in Swift. We've introduced many features to help protect mutable data. Actors are a good default go-to solution for protecting mutable state because it isolates the stored data in its own domain. At any given point in time, only one task will be executing "on" the actor, and have exclusive access to it. Multiple tasks cannot access state protected by the actor at the same time, although they may interleave execution at potential suspension points (indicated by `await`). In general, the actor approach also leans itself well to code organization, since the actor's state, and operations on this state are logically declared in the same place: inside the actor.
```